templates/data/istio-module-data.yaml

```diff
@@ -196,6 +196,13 @@ templates:
               - jobConfig:
                   name: nightly-istio-performance-test-gcp
                   cron: "0 0 * * *"
+                  path_alias: github.com/kyma-project/istio
+                  extra_refs:
```

Comment:
Why do we need the extra_refs with the same path_alias as in the jobConfig? Shouldn't it be enough to only have the path_alias in jobConfig?